src/validation.cpp

```diff
@@ -457,10 +457,12 @@ class MemPoolAccept
         std::vector<COutPoint>& m_coins_to_uncache;
         /** When true, the transaction or package will not be submitted to the mempool. */
         const bool m_test_accept;
-        /** Whether we allow transactions to replace mempool transactions by BIP125 rules. If false,
+        /** Whether we allow transactions to replace mempool transactions. If false,
          * any transaction spending the same inputs as a transaction in the mempool is considered
          * a conflict. */
         const bool m_allow_replacement;
+        /** When true, allow sibling eviction. This only occurs in single transaction package settings. */
+        const bool m_allow_sibling_eviction;
```

Comment:
I thought sibling eviction was relevant in a TRUC Transaction constellation when there already exist an unconfirmed parent and unconfirmed child, and now a newly submitted transaction is a child of the same parent, but spends a different output than the original child, i.e. not actually in conflict with the original child except for the topological restrictions affecting TRUC Transactions.

If I got all that right, I would posit that "This only occurs in single transaction package settings." is a bit confusing, unless I’m the only person for whom it’s not obvious that "single transaction package settings" is exclusively describing the replacement transaction.

Reply:
"single transaction subpackage"?

Reply:
> "single transaction package settings" is exclusively describing the replacement transaction.

The replacement transaction is always going to be the one we're currently validating - maybe I'm misunderstanding what you're saying :sweat_smile: . I agree "single transaction package" doesn't mean much to me in this context, perhaps "disallowed when evaluating a multi-transaction subpackage or package" ?